pandas/tests/io/test_excel.py

```diff
@@ -844,7 +841,7 @@ def test_read_excel_squeeze(self, ext):
         tm.assert_series_equal(actual, expected)
 
 
-@td.skip_if_no('xlrd', '1.0.0')
+@td.skip_if_no('xlrd')
```

Comment:
Comment might have been lost on a prior PR but the min xlrd version for pandas is 1.0.0 anyway, so this was duplicative